packages/client/src/exampleTypes.js

```diff
@@ -360,4 +360,87 @@ export default F.stampKey('type', {
       search: null,
     },
   },
+  dateIntervalGroupStats: {
+    reactors: {
+      groupField: 'self',
```

Comment:
For my education, looking at the file I usually see `self`, `others` and `all` as valid values, what do they mean?

Reply:
These values tell contexture-client what to do when that property changes
- `self` means that changing the property should mark the contexture node the property belongs to as "dirty"
- `others` means that changing the property should mark all other contexture nodes that depend on this one as "dirty"
- `all` is a combination of both `self` and `others`

If this all sounds confusing then you're not alone. It's not a complicated mental model, we just don't have TypeScript types or good enough documentation in these packages today.

Reply:
Love it! Could you clarify what do you mean by "dirty", or, when can we consider a node is "dirty"?

Reply:
Dirty means that we need to call the backend to get results for that node. The contexture-client library can either submit debounced backend requests when nodes are marked as "dirty" or it can wait for the user to click "Search" to send the request.